packages/artifact_proxy/lib/config.dart

```diff
@@ -22,6 +26,7 @@ final engineArtifactPatterns = {
   r'flutter_infra_release\/flutter\/(.*)\/ios-release\/artifacts\.zip',
   r'flutter_infra_release\/flutter\/(.*)\/ios-profile\/artifacts\.zip',
   r'flutter_infra_release\/flutter\/(.*)\/flutter-web-sdk-darwin-x64\.zip',
+  r'flutter_infra_release\/flutter\/(.*)\/flutter-web-sdk-windows-x64\.zip',
```

Comment:
We're presumably missing the -linux variant of this too. ☹️

Reply:
:(